x-pack/legacy/plugins/task_manager/task_manager.ts

```diff
@@ -127,6 +127,20 @@ export class TaskManager {
     startPoller();
   }
 
+  private async claimAvailableTasks() {
+    const { docs, claimedTasks } = await this.store.claimAvailableTasks({
+      size: this.pool.availableWorkers,
+      retryAt: intervalFromNow('1s')!,
```

Comment:
I'm not sure if `1s` is the best timeout to give for claiming tasks. I'm thinking under stress in Elasticsearch and Kibana, each query could take a few seconds or a lot more to finish (`updateByQuery` then `update` / `bulkUpdate`). A low timeout would cause other servers to attempt to claim the task as well.

I'm thinking we set a realistic timeout we believe would make sense for another server to try and claim the task. Mostly in scenario the server went down. Something like `30s` or `1m` maybe? And we can possibly make this configurable too.

Reply:
Oh, absolutely, that was just to get the to a point where I can test it.
It's now at 30 seconds and running two Kibana side by side which seemed to be good enough for now (I log at `debug` when you are late to set status as `running`, so we can see how it performs in tests).

I believe we'll be able to dial this much lower once the `markTaskAsRunning` is better paralelised which is my next task.